packages/core/chrome/core-chrome-browser-internal/src/ui/header/header_help_menu.tsx

```diff
@@ -159,6 +159,21 @@ export class HeaderHelpMenu extends Component<Props, State> {
 
         <EuiSpacer size="xs" />
 
+        <EuiButtonEmpty
```

Comment:
Is it possible to hide this link on cloud? @yuliacech maybe you can help @claracruz take a look based on the work that was done in https://github.com/elastic/kibana/pull/143765?